openfisca_us/variables/demographic/spm_unit.py

```diff
@@ -41,8 +41,27 @@ class spm_unit_assets(Variable):
     definition_period = YEAR
 
 
+class spm_unit_income(Variable):
```

Comment:
Remove this and use a `ccdf_income` variable in the `ccdf` folder (every program will have a different income variable, since definitions can vary).